app/models/rating.rb

```diff
@@ -13,13 +5,5 @@ class Rating < ActiveRecord::Base
   validates :user, presence: true
   validates :skill, presence: true
   validates :score, inclusion: { in: 0..4 }
-  validate :cannot_have_the_same_user_and_skill_as_another_rating
-
-  private
-
-  def cannot_have_the_same_user_and_skill_as_another_rating
-    if self.class.where(user: user, skill: skill).first
-      errors[:base] << "Rating already exists for this user and skill."
-    end
-  end
+  validates :user_id,  :uniqueness => {:scope => [:skill_id]}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.